spring-integration-core/src/main/java/org/springframework/integration/support/management/TimerFacade.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.support.management;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Gary Russell
+ * @since 5.0.4
+ *
+ */
+public interface TimerFacade {
+
+	SampleFacade start(MetricsCaptor captor);
```

Comment:
Confused; your earlier comment:

>This API is somehow is not consistent with the Timer.start(this.meterRegistry).
>Maybe we can do something similar as well like TimerFacade.start(this.metricsCaptor)?

I couldn't use `TimerFacade.start()` in the caller because it has no knowledge of a concrete implementation. But, ok; I see what you mean.

Reply:
Right, I also learn the code during review. This is fully new matter for me as well.

So, sorry for confusing.